docs/notebooks/util/plotting.py

```diff
@@ -133,6 +133,40 @@ def plot_function_2d(
     return fig, ax
 
 
+def plot_acq_function_2d(
+    acq_func,
+    mins: TensorType,
+    maxs: TensorType,
+    grid_density=20,
```

Comment:
Other types would be good here as well as in the original plot_function_2d, to help catch e.g. somebody updating one function and not the other.

```suggestion
    grid_density: int = 20,
```

(An alternative would have been to declare the function as `def plot_acq_function_2d(acq_func, *args, **kwargs)` and pass all the args and kwargs to `plot_function_2d`, which is referenced explicitly in the docstring. The downside is that you have to look up plot_function_2d to figure out how to use plot_acq_function_2d.)